projects/syntheticBabylon/index.md

```diff
@@ -0,0 +1,103 @@
+---
+authors:
+  - Theodor Hillmann
+authors-url:
+  - "http://theodorhillmann.de"
+layout: project
+title: "Synthetic Babylon: analogue generative architecture"
+type: project
+---
+
+# Synthetic Babylon
+## analogue generative architecture
+
+This work is part of a class at the University of Applied Sciences Potsdam. The project engages in analogue generative design. Initially an analogue algorithm has been written. I tried to generate architectural sketches inspired by Constant's "New Babylon". This initially analog algorithm has been extended and developed over many iterations and finally put in form.
+
+For the algorithms first iteration pencils and sheets of paper were handed to 5 people. They were instructed to consequently follow the rules of the algorithm listed below. The following "code" is kept in written-german to stay closest to the original experience.
+
+### Iteration 1:
+
+Arbeite die folgenden Punkte nacheinander ab.
+Nutze dafür ein blankes Blatt Papier und ein Bleistift (wenn
+möglich).
+
+1. Zeichne zwei Punkte die waagerecht zueinander stehen, und
+nicht weiter entfernt sind als ein Finger lang ist.
+2. Zeichne unterhalb – zwischen diesen Punkten eine Senkrechte,
+kaum länger als ein Fingernagel.
+3. Verbinde die Enden der Senkrechten jeweils mit den beiden
+Punkten durch gerade Linien.
+4. Zeichne eine beliebige Anzahl von Senkrechten zwischen diese
+Schrägen.
+5. Verbinde die entstandenen oberen Berührungspunkte auf den
+Schrägen mit den noch nicht verbundenen Punkt.
+6. Setze diese Technik fort bis ein Gefühl von Stadtraum entsteht
+
+Vielen Dank!
+
+The results turned out to be rough and did not fulfil the visual expectations.
+Therefore in the following steps the algorithm has continuously been adjusted to the impressions of natural and organic structures.
+
+![](https://github.com/t0dorakis/syntheticBabylon/blob/master/assets/visuals/IT1/1.png)
```

Comment:
No absolut links to your repo please. When you updated the folder structure you can link them like this:  

    ![](./assests/images/it1-1.jpg)